<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KNHDASJ" ignored="0" name="Test" tests="3" failures="1" timestamp="2022-01-18T18:49:15 IST" time="23.098" errors="0">
  <testcase name="validLogin" time="5.126" classname="com.extent.test.OrangeHRMTestEXTENT"/>
  <testcase name="logOut" time="4.286" classname="com.extent.test.OrangeHRMTestEXTENT"/>
  <testcase name="invalidLogin" time="2.066" classname="com.extent.test.OrangeHRMTestEXTENT">
    <failure type="java.lang.AssertionError" message="expected [Invalid credential] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: expected [Invalid credential] but found [Invalid credentials]
at com.extent.test.OrangeHRMTestEXTENT.invalidLogin(OrangeHRMTestEXTENT.java:67)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
</testsuite> <!-- Test -->
